app/assets/src/components/ui/controls/dropdowns/BareDropdown.jsx

```diff
@@ -227,6 +229,7 @@ class BareDropdown extends React.Component {
         {...baseDropdownProps}
         className={dropdownClassName}
         onBlur={e => e.stopPropagation()}
+        disableAutocomplete={disableAutocomplete}
```

Comment:
I don't think semantic-ui's Dropdown accepts disableAutocomplete. If not, maybe remove this?

Reply:
Thanks! not sure why I added it there..